serializer/resources/org/apache/xml/serializer/Version.src

```diff
@@ -1,150 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements. See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership. The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the  "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-/*
- * $Id$
- */
-package org.apache.xml.serializer;
-
-/**
- * Administrative class to keep track of the version number of
- * the Serializer release.
- * <P>This class implements the upcoming standard of having
- * org.apache.project-name.Version.getVersion() be a standard way 
- * to get version information.</P> 
- * @xsl.usage general
- */
-public final class Version
```

Comment:
Does this PR remove a public class? If so, that's a -1 for breaking binary compatibility.
Let's make sure the Maven build uses JApiCmp to check.

Reply:
Would you mind reviewing instead of making premature assumptions? Thank you.

Reply:
Hello @kriegaex 
My review is asking the question: Does this PR break binary compatibility? 

There is no sense in spending time looking at details unless we are standing on solid ground.

Reply:
That is not a review, it is a cross-examination. If you feel that you are too busy to conduct a proper review, I suggest you better do something that has higher priority to you. I understand that nobody can be in several places simultaneously. Either way, I think I am too busy to answer this kind of question. You would already know the answer, if you had taken even a cursory glance at the list of changed files or read the PR description.